src/client.rs

```diff
@@ -1,14 +1,17 @@
 use libc::{c_char, size_t};
+use std::io::ErrorKind::ConnectionAborted;
 use std::io::{BufReader, Cursor, Read, Write};
```

Comment:
`cargo fmt` is fine with the older version, and I prefer it to your proposal.

Reply:
Fair enough, anything that's good enough for `cargo fmt` is good enough for me.

Reply:
My personal style is: no nested qualifiers in `use` statements. Or another way of putting it: A `use` statement should end in at most one set of braces, and those braces should not contain any `::`.

I sometimes deviate from this ideal by accident, since I let rust-analyzer fill in my `use` statements for me, and it is not opinionated in the same way. In general `rustfmt` seems to allow a lot of variation here and not try very hard to canonicalize.